### ISSUE TITLE
Different languages for the Wiki`s

### ISSUE THREAD

**Issue by ShalokShalom** [go-gitea/gitea#1879]
- Gitea version (or commit ref): All
- Git version: All
- Operating system: All
- Database (use `[x]`):
  - [x] PostgreSQL
  - [x] MySQL
  - [x] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (https://try.gitea.io/ShalokShalom/start/wiki/Installation-)

## Description

Hi there ^-^

I like to provide the Wiki and Issues section in different languages, so that the content of it changes with the choosen UI language.

**Comment by lunny** [comment]
Do you mean multi-language support of Wiki? So that one page could has a language property?

**Comment by ShalokShalom** [comment]
Yes and the same with the Issues page. :)

**Comment by bkcsoft** [comment]
How would that work for issue pages?

**Comment by ShalokShalom** [comment]
For different issues in different languages.

**Comment by bkcsoft** [comment]
Maybe I don't understand the use case here. So a bug-report written in English does not apply to Russian users? 

As for wiki, that already supports directories, so spliting that up into folders should be fairly straight forward.

**Comment by ShalokShalom** [comment]
Yeah exactly. I want to use the issue pages for some kind of a user forum.

**Comment by ShalokShalom** [comment]
Is this possible?

**Comment by lunny** [comment]
It's possible if we add `<repo_name>.en.wiki.git`, `<repo_name>.zh.wiki.git` and add a switch on the UI.

**Comment by bkcsoft** [comment]
@lunny That would work for wikis yes, and quite easily. But for issues it's a bit more complicated.

**Comment by lunny** [comment]
@bkcsoft yes.  @ShalokShalom I think this could be split as two issues so that someone could send PR to resolve wiki issue at first.

**Comment by ShalokShalom** [comment]
Sure. ^-^ 

This is the issues issue: https://github.com/go-gitea/gitea/issues/3409
And this one here is the Wiki`s issue ^-^

**Comment by ShalokShalom** [comment]
https://github.com/go-gitea/gitea/issues/3409

**Comment by ShalokShalom** [comment]
Is this still on the table?

**Comment by lafriks** [comment]
If someone creates PR for this I don't see reason why it would not be :)
